clients/web/index.html

```diff
@@ -6,11 +6,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Tauri Devtools Web-Client</title>
   </head>
-  <body class="bg-navy-700 text-white">
+
+  <body style="background-color: #070c17; color: #fff">
```

Comment:
@alve-crabnebula Could you explain this change to me?

Reply:
It's to prevent the "white flash" on first loading the app, as the Tailwind classes take a bit to load, so having this part as inlined CSS on the very first request means the browser can render a proper background before even making a request to load Tailwind

At least locally It provides a smoother experience